roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -58,6 +58,8 @@ auth:
   openshift:
     realm: openshift
 middleware:
+  registry:
+  - name: openshift
```

Comment:
After discussing with @miminar this config is only valid for 2.4.1 registry, so we need to make this conditional at least on 3.3 for now. Wrap this in 
```
{% if openshift.common.version_gte_3_3_or_1_3 | bool %}
{% endif -%}
```